clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/description/UpsertGoogleLoadBalancerDescription.groovy

```diff
@@ -26,6 +29,8 @@ class UpsertGoogleLoadBalancerDescription extends AbstractGoogleCredentialsDescr
   String portRange
   String region
   String accountName
+  GoogleHttpLoadBalancer googleHttpLoadBalancer
```

Comment:
Feels a bit weird to have this as an attribute since it contains some of the same attributes that are already siblings of `googleHttpLoadBalancer`. Haven't thought very deeply about it, but would have expected this description to have as top-level elements the full set of load balancer attributes; and then some subset would apply to each type of load balancer.